edit-credential-modal-directive.js

```diff
@@ -39,7 +38,12 @@ function factory(brAlertService, brCredentialService) {
       }
       model.loading = true;
       brAlertService.clearFeedback();
-      brCredentialService.collection.update(credential)
+      var options = {};
+      var search = $location.search();
+      if('id' in search) {
+        options.url = $location.path() + '?id=' + search.id;
+      }
```

Comment:
I think this is ok for now, but we should add a TODO that says the update URL should be passed into the directive. We don't want to do any sort of magic here.